experience.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Thom Yorke Resume</title>
+    <link rel="stylesheet" type="text/css" href="css/style.css">
+  </head>
+  <body>
+    <h1>Thom Yorke</h1>
+    <nav>
+      <a class="nav" href="index.html">bio</a> | 
+      <a class="nav" href="experience.html">experience</a>
+    </nav>
+    <hr>
+    <center>
```

Comment:
@jcooper8190 it works but going forward lets use css to achieve the same results.